src/patternfly/components/Wizard/wizard.scss

```diff
@@ -59,7 +64,6 @@
 
   // Toggle list
   --pf-c-wizard__toggle-list--MarginRight: var(--pf-global--spacer--sm);
-  --pf-c-wizard__toggle-list--MarginBottom: calc(var(--pf-c-wizard__toggle-list-item--MarginBottom) * -1);
```

Comment:
Did you mean to remove this? Seems like you want to just move it below where `--pf-c-wizard__toggle-list-item--MarginBottom` is defined.

Reply:
I did mean to remove it because this one also came out to be zero.  I'll double check to make sure.

Reply:
it's `-4px` for me.

Reply:
yeah my mistake thanks will add it back.

Reply:
Done.